### ISSUE TITLE
Pika stripping comments inside strings

### ISSUE THREAD

**Issue by chowey** [pikapkg/cdn#22]
This shows up in [es-module-shim](https://cdn.pika.dev/-/es-module-shims@v0.4.7-iVxJEhO4aDmWgEVhkYjN/dist=es2019/es-module-shims.js), line 2343:

```js
    const sourceMappingIndex = resolvedSource.lastIndexOf('
```

Compare to the [source file](https://github.com/guybedford/es-module-shims/blob/0.4.7/src/es-module-shims.js#L120):

```js
    const sourceMappingIndex = resolvedSource.lastIndexOf('//# sourceMappingURL=');
```

Everything after the "comment" is stripped from the pika build, even though the "comment" is inside a string.

**Comment by FredKSchott** [comment]
Good catch. That's correct: our current sourceMappingURL strip logic is regex based, so it can't tell the different between a string and a comment.

+1 for fixing this!

**Comment by chowey** [comment]
No problem!

> +1 for fixing this!

I don't quite understand - are you saying you want a patch? Where is the source code?

**Comment by FredKSchott** [comment]
Source code isn't public, my comment was meant to mean that  we'll prioritize fixing this as soon as we can.

**Comment by drwpow** [comment]
We have an open PR to fix this issue on ~~snowpack~~ esinstall (our builder): https://github.com/pikapkg/snowpack/pull/1146. Once this goes into a Snowpack release, we’ll update the CDN soonafter.

**Comment by drwpow** [comment]
Update on this: we are no longer stripping out the comments for `es-module-shims`, but it looks like we still have a build error on that package due to its reliance on web assembly. Currently looking into that.

**Comment by ziadkh0** [comment]
@drwpow Is there an issue tracking that build error?
